homeassistant/components/lutron_caseta/manifest.json

```diff
@@ -10,5 +10,9 @@
   },
   "codeowners": ["@swails", "@bdraco", "@danaues"],
   "iot_class": "local_push",
-  "loggers": ["pylutron_caseta"]
+  "loggers": ["pylutron_caseta"],
+  "supported_brands": {
+    "lutron_caseta_pro": "Lutron Caséta Pro",
```

Comment:
This is adding a product? That is not what supported brands as for.

Reply:
Lutron has a confusing branding strategy.  All their system are intentionally incompatible with each other to protect their installer and avoid canalizing their high end market.

https://github.com/home-assistant/core/pull/77425#discussion_r957658717

Reply:
> Lutron has a confusing branding strategy. All their system are intentionally incompatible with each other to protect their installers revenue stream and avoid market cannibalization of their high end market.

This is incompatible with what "brand" entails in Home Assistant. A brand is a manufacturer, in this case Lutron, not a product or integration name.

Reply:
Lutron treats each of their lines as separate brands, with separate site, separate installers, separate compatibility.

But Lutron brands != Home Assistant brands so I see you point.  I'll close these PRs